Disable extract hash commit from build as it expect
using a git repository which is not our use-case.

The original commit used to determine what to disable:
https://github.com/intel/vc-intrinsics/commit/34807698d80e34a81acf14d5aad28eb088d8c0b7

Current issue opened in hope to get a permanent fix:
https://github.com/intel/vc-intrinsics/issues/130

--- GenXIntrinsics/include/llvm/GenXIntrinsics/CMakeLists.txt-orig	2024-07-06 00:27:35.000000000 +0000
+++ GenXIntrinsics/include/llvm/GenXIntrinsics/CMakeLists.txt	2024-07-23 00:23:26.694931271 +0000
@@ -18,44 +18,8 @@ add_custom_command(
     COMMENT "Building ${GENX_INTRINSICS_DESCRIPTION}..."
 )
 
-find_first_existing_vc_file(intrinsics_vc "${GENX_INTRINSICS_MAIN_DIR}")
-
-set(version_inc ${CMAKE_CURRENT_BINARY_DIR}/GenXVersion.inc)
-
-if(LLVM_GENX_INTRINSICS_ROOT_DIR)
-  set(LLVM_GENX_INTRINSICS_REPO_DIR ${LLVM_GENX_INTRINSICS_ROOT_DIR})
-else()
-  set(LLVM_GENX_INTRINSICS_REPO_DIR ${GENX_INTRINSICS_MAIN_DIR}/..)
-endif()
-
-if(${LLVM_VERSION_MAJOR} LESS 9)
-  find_file(FOUND_VCS GetSVN.cmake PATHS ${LLVM_CMAKE_DIR} REQUIRED)
-  add_custom_command(
-    OUTPUT "${version_inc}"
-    COMMAND
-    ${CMAKE_COMMAND} "-DSOURCE_DIRS=${LLVM_GENX_INTRINSICS_REPO_DIR}"
-                     "-DNAMES=VCI"
-                     "-DHEADER_FILE=${version_inc}"
-                     -P "${FOUND_VCS}")
-else()
-  find_file(FOUND_VCS VersionFromVCS.cmake PATHS ${LLVM_CMAKE_DIR} REQUIRED)
-  add_custom_command(
-    OUTPUT "${version_inc}"
-    COMMAND
-    ${CMAKE_COMMAND} "-DSOURCE_DIR=${LLVM_GENX_INTRINSICS_REPO_DIR}"
-                     "-DNAME=VCI"
-                     "-DHEADER_FILE=${version_inc}"
-                     "-DVCS_SCRIPT=${FOUND_VCS}"
-                     -P "${CMAKE_CURRENT_SOURCE_DIR}/ConfigureVersionFile.cmake")
-endif()
-
-set_source_files_properties("${version_inc}"
-    PROPERTIES GENERATED TRUE
-    HEADER_FILE_ONLY TRUE)
-
 add_custom_target(GenXIntrinsicDescriptionGen
     DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/${GENX_INTRINSICS_DESCRIPTION}
-    ${version_inc}
 )
 add_custom_target(GenXIntrinsicsGen)
 add_dependencies(GenXIntrinsicsGen GenXIntrinsicDescriptionGen)
--- GenXIntrinsics/lib/GenXIntrinsics/GenXVersion.cpp-orig	2024-07-06 00:27:35.000000000 +0000
+++ GenXIntrinsics/lib/GenXIntrinsics/GenXVersion.cpp	2024-07-23 01:16:10.514186559 +0000
@@ -9,7 +9,6 @@ SPDX-License-Identifier: MIT
 // This source file defines interface functions to retrive version info.
 
 #include "llvm/GenXIntrinsics/GenXVersion.h"
-#include "llvm/GenXIntrinsics/GenXVersion.inc"
 
 std::string llvm::GenXIntrinsic::getVCIntrinsicsRevision() {
 #ifdef VCI_REVISION
